packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -439,22 +441,32 @@ async function fetchLatestSnapshotCore(
                     // Time from the end of the connection until the inital handshake process to secure the connection.
                     secureConnectionTime,
                     // Interval between the initial fetch until the last byte is received.
+                    // Likely same as fetchTime + receiveContentTime.
                     fetchStartToResponseEndTime,
+                    // reqStartToResponseEndTime = fetchStartToResponseEndTime - <initial TCP handshake>
                     // Interval between starting the request for the resource until receiving the last byte.
                     reqStartToResponseEndTime,
+                    // networkTime = fetchStartToResponseEndTime - sprequestduration
                     // Interval between starting the request for the resource until receiving the last byte but
-                    // excluding the Snaphot request duration indicated on the snapshot response header.
+                    // excluding the snaphot request duration indicated on the snapshot response header.
                     networkTime,
-                    // Measures total time to make fetch call. Should be similar to networkTime + sprequestduration.
+                    // Measures time to make fetch call. Should be similar to
+                    // fetchStartToResponseEndTime - receiveContentTime, i.e. it looks like it's time till first byte /
+                    // end of response headers
                     fetchTime,
```

Comment:
I think it should be similar to fetchStartToResponseEndTime . receiveContentTime is executed later on after receiving the response?

Reply:
Take a look at this screen shot from https://dev.azure.com/fluidframework/internal/_workitems/edit/2548 - it clearly does not. SnapshotParse starts way early, before we have getLatest completed. Also (not captured in the picture), callstack shows that actual parsing happens only after getLatest is over. This all points out that fetch() returns way before we have any content. I think the way it works (and maybe it's browser specific) is that fetch() returns when all response headers are in, but payload is not yet in. Thus the comment

![image](https://user-images.githubusercontent.com/46826984/201429550-defd9b26-f970-46d3-8ad2-a308763db63f.png)